### PR TITLE
New package: speedtest-cli-1.0.0

### PR DESCRIPTION
**Pull request by drkhsh**

### CODE REVIEW

File: srcpkgs/speedtest-cli/template [hunk 1]
```diff
@@ -0,0 +1,13 @@
+# Template file for 'speedtest-cli'
+pkgname=speedtest-cli
+version=1.0.0
+revision=1
+build_style=python3-module
+makedepends="python3-setuptools"
```
Comment (Vaelatern) [inline srcpkgs/speedtest-cli/template]: I do think you mean `hostmakedepends` here

File: srcpkgs/speedtest-cli/template [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+depends="python3-setuptools"
```
Comment (Vaelatern) [inline srcpkgs/speedtest-cli/template]: You sure you depend on that after build time?

File: srcpkgs/speedtest-cli/template (hunk 1 above)
Comment (drkhsh) [inline srcpkgs/speedtest-cli/template]: sure

File: srcpkgs/speedtest-cli/template (hunk 2 above)
Comment (drkhsh) [inline srcpkgs/speedtest-cli/template]: i dont think so, i will test

**Comment by drkhsh** [comment]
@Vaelatern please check now

**Pull request merged**